src/sqlfluff/core/dialects/dialect_snowflake.py

```diff
@@ -112,6 +112,13 @@
         Ref("QuotedIdentifierSegment"),
         Ref("ColumnIndexIdentifierSegment"),
     ),
+    PostFunctionGrammar=Sequence(
+        Ref("WithinGroupClauseSegment", optional=True),
+        Sequence(
+            Sequence(OneOf("IGNORE", "RESPECT"), "NULLS", optional=True),
+            Ref("OverClauseSegment"),
+        ),
+    ),
```

Comment:
Does this need to be a nested set of sequences?

I think this would be easier to read, and functionally identical:

```
Sequence(
    Ref("WithinGroupClauseSegment", optional=True),
    Sequence(OneOf("IGNORE", "RESPECT"), "NULLS", optional=True),
    Ref("OverClauseSegment"),
),
```

Reply:
Ah yes a remnant from the copy over from the postgres dialect. Good catch